src/libpriv/rpmostree-passwd-util.c

```diff
@@ -1182,10 +1057,7 @@ rootfs_has_usrlib_passwd (int rootfs_dfd,
           return TRUE;
         }
       else
-        {
-          glnx_set_error_from_errno (error);
-          return FALSE;
-        }
+        return glnx_throw_errno_prefix (error, "fstatat");
```

Comment:
Yup, that works! (Will review there). I'll admit I wasn't looking at the possible reasons for failures, just want to avoid things like #845, #846 later on. Upstreaming it in libglnx wrappers is a good idea.